src/main/java/net/minecraftforge/client/ForgeHooksClient.java

```diff
@@ -389,9 +391,9 @@ public static float getFogDensity(FogMode type, Camera camera, float partialTick
         return -1;
     }
 
-    public static void onFogRender(FogMode type, Camera camera, float partialTick, float distance)
+    public static void onFogRender(FogMode type, Camera camera, float partialTick, float nearDistance, float farDistance, FogShape shape)
     {
-        MinecraftForge.EVENT_BUS.post(new EntityViewRenderEvent.RenderFogEvent(type, camera, partialTick, distance));
+        MinecraftForge.EVENT_BUS.post(new EntityViewRenderEvent.RenderFogEvent(type, camera, partialTick, nearDistance, farDistance, shape));
     }
```

Comment:
This still is counted as a breaking change to my understanding, and therefore must be treated appropriately (terminally deprecated, overload with new parameters).